streams/src/main/java/org/apache/kafka/streams/errors/InvalidStateStoreException.java

```diff
@@ -18,13 +18,8 @@
 
 
 /**
- * Indicates that there was a problem when trying to access a
- * {@link org.apache.kafka.streams.processor.StateStore StateStore}, i.e, the Store is no longer valid because it is
- * closed or doesn't exist any more due to a rebalance.
- * <p>
- * These exceptions may be transient, i.e., during a rebalance it won't be possible to query the stores as they are
- * being (re)-initialized. Once the rebalance has completed the stores will be available again. Hence, it is valid
- * to backoff and retry when handling this exception.
+ * <p>Indicates that there was a problem when trying to access a {@link org.apache.kafka.streams.processor.StateStore StateStore}.
+ * {@code InvalidStateStoreException} not thrown directly but only following sub-classes.
```

Comment:
Nit: `Note that {@code InvalidStateStoreException} [is] not thrown directly but only [its] sub-classes.`